packages/metatransaction-broadcaster/MetatransactionBroadcaster.js

```diff
@@ -234,6 +234,25 @@ class MetatransactionBroadcaster {
       // Not a voting rep related transaction (we recognise)
     }
 
+    const multicallDef = await this.loader.load({ contractName: "Multicall" }, { abi: true, address: false });
+    const possibleMulticall = new ethers.Contract(target, multicallDef.abi, this.wallet);
+
+    try {
+      const tx = possibleMulticall.interface.parseTransaction({ data: txData });
+      if (tx.signature === "multicall(bytes[])") {
+        // We check for each multicall whether it's doing something we'd allow
+        for (let i = 0; i < tx.args[0].length; i += 1) {
+          const action = tx.args[0][i];
+          const valid = await this.isColonyFamilyTransactionAllowed(target, action, userAddress);
+          if (!valid) {
+            return false;
+          }
+        }
+      }
+    } catch (err) {
```

Comment:
Under what circumstance would we `err` here? E.g. why is this wrapped in try/catch?

Reply:
When it's not a multicall transaction? I think I'm not understanding the intent of the question :sweat_smile:

Reply:
Curious which line would throw the exception. Is it this line `const tx = possibleMulticall.interface.parseTransaction({ data: txData });`?

Reply:
Yep - if it's not a multicall transaction, then that interface cannot parse the transaction data and throws an error.

Reply:
In that case, can we limit the try/catch to just that line? Wrapping the entire section in a try/catch suggests that other parts of the statement could throw.

Reply:
We don't want any code in the try to run if that line errors though. Are you looking for something like:
```
try {
   statementThatMightThrow
} catch (err) {
   error = err
}
if (!error) {
   rest of Code
}
```

I'll also note that this exact pattern is used twice more in the file; am I sorting those out too?